doc/schema/schema.md

```diff
@@ -68,16 +68,18 @@ Samples from the posterior distribution p(theta|y).
 ### `sample_stats`
 Information and diagnostics for each `posterior` sample, provided by the inference backend. It may vary depending on the algorithm used by the backend (i.e. an affine invariant sampler has no energy associated). The name convention used for `sample_stats` variables is the following:
 * `lp`: (unnormalized) log probability for sample
-* `step_size`
-* `step_size_bar`
+* `step_size`: the step size used for this sample
+* `step_size_bar`: the current best known step-size
 * `tune`: boolean variable indicating if the sampler is tuning or sampling
-* `depth`:
-* `tree_size`:
-* `mean_tree_accept`:
+* `depth`: HMC-NUTS only, the depth of the tree that was used to generate this sample
+* `tree_size`: HMC-NUTS only, the number of leafs of the sampling tree, when the sample was accepted
+* `n_leapfrog`: HMC-NUTS only, number of steps selected in each iteration to traverse the posterior
```

Comment:
Could you elaborate on the relationship between these? I know in the past when I ran this done, the value of one bounded the others (e.g. I believe `2^depth` bounds `tree_size` and `n_leapfrog`, but how does `tree_size` relate to `n_leapfrog`?) and it would be good while defining these to explain those constraints.